__tests__/components/navigation.js

```diff
@@ -53,31 +90,7 @@ describe("Navigation Component", () => {
     });
 
     it("should render a google login button if the client does not have a session cookie", async () => {
```

Comment:
nit: The name of this test is likely no longer accurate since we don't really do any explicit cookie management anymore.